DOL.WHD.Section14c.Web/src/modules/components/dolHeader/dolHeaderDirective.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function(ngModule) {
+  ngModule.directive('dolHeader', function() {
```

Comment:
Oh neat.  Because the AngularJS directive has the same name as the Angular 4 component, there's no need to change anywhere that it's being used, right?  Very handy.

Reply:
@mgwalker correct!